domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/AktørYtelse.java

```diff
@@ -165,7 +165,16 @@ void leggTilYtelse(Ytelse ytelse) {
     }
 
     void tilbakestillYtelser() {
-        this.ytelser.clear();
+        var legacy = ytelser.stream()
```

Comment:
Fungerer greit det her.

Kunne også vært:
`this.ytelser = ytelser.stream()
    .filter(AktørYtelse::beholdLegacyYtelseFraKilde)
    .collect(Collectors.toSet());`